dapps/react-dapp-v2/src/contexts/JsonRpcContext.tsx

```diff
@@ -292,10 +296,10 @@ export function JsonRpcContextProvider({
 
         // encode message (hex)
         const hexMsg = encoding.utf8ToHex(message, true);
-
+        const hashMsg = hashPersonalMessage(message)
```

Comment:
Since this is only used inside `verifySignature` at the bottom, can we call `hashPersonalMessage` inline there pls 🙏